js/models/listing/ListingShort.js

```diff
@@ -22,6 +22,15 @@ export default class extends BaseModel {
     return this.get('shipsTo').indexOf(country) !== -1;
   }
 
+  parse(response) {
+    const parsedResponse = response;
```

Comment:
There's no point in setting parsedResponse to response. Objects are assigned by reference, so later when you change parsedResponse, you're also changing response.

If you want to modify a separate object (at least at the top level), then you would want:

```
const parsedResponse = { ...response };
```

Otherwise, just modify response directly.